dts/bindings/counter/nxp,lptmr.yaml

```
@@ -20,7 +20,13 @@ properties:
   clk-source:
     type: int
     required: true
-    description: Prescaler clock source (0 to 3)
+    description: |
```

```suggestion
    enum: [0, 1, 2, 3]
    description: |
```
To have the check done at build-time rather than only described.